src/tokens.ts

```diff
@@ -0,0 +1,22 @@
+import { Token } from '@lumino/coreutils';
+import { ICreateJobModel, JobsView } from './model';
+
+namespace Scheduler {
+
+  export type EnvironmentParameterValue = string | number | boolean
+
+  export interface IAdvancedOptionsProps {
+    mode: JobsView
+    model: ICreateJobModel
```

Comment:
This is likely going to be support in addition to `ICreateJobModel`, an analogous to the `IJobDetailModel` with all properties from the `IDescribeJob`.